src/compose/application-manager.ts

```diff
@@ -680,9 +680,9 @@ function saveAndRemoveImages(
 				}) ?? _.find(availableImages, { dockerImageId: svc.config.image }),
 		),
 	) as imageManager.Image[];
-	const targetImages = _.flatMap(target, (app) =>
-		_.map(app.services, imageForService),
-	);
+
+	const targetServices = _.flatMap(target, (app) => app.services);
```

Comment:
```suggestion
	const targetServices = target.flatMap(app => app.services);
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap

Reply:
We need to upgrade to `es2019` in `tsconfig.json` to be able to use those operators

Reply:
We need to set `es2019` on `tsconfig.json` for that.

Reply:
We need to add `es2019` to `tsconfig.json` for that to work

Reply:
I thought it would be painful but it was straightforward, I'll add it to the PR